lib/observe.dart

```diff
@@ -7,19 +7,15 @@ library observe;
 // This library contains code ported from observe-js:
 // https://github.com/Polymer/observe-js/blob/0152d542350239563d0f2cad39d22d3254bd6c2a/src/observe.js
 // We port what is needed for data bindings. Most of the functionality is
-// ported, except where differences are needed for Dart's Observable type.
+// ported, except where differences are needed for Dart's AutoObservable type.
 
+export 'package:observable/observable.dart'
```

Comment:
consider: re-exporting ObservableList/ObservableMap as well? maybe to_observable stuff?

might make it easier for folks to upgrade.